src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -212,6 +209,12 @@
     <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.HandleOptions.cs">
       <Link>Common\Interop\Windows\Interop.ProcessOptions.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\System\Text\EncodingHelper.cs">
```

Comment:
Is this file Windows specific?  I see one next to it with the .Unix.cs suffix, but this one doesn't have a suffix.